libs/components/modals/src/lib/modules/confirm/confirm.component.ts

```diff
@@ -55,6 +58,8 @@ export class SkyConfirmComponent {
 
     this.message = config.message;
     this.body = config.body;
+    // Using the service here instead of the directive due to the body element being conditional and thus a template variable being unavailable at an outer scope
```

Comment:
Can we reword to `...due to the confirm component's "body" container being conditional...`? I was initially confused that this comment was talking about the `<body>` element.

Reply:
Sure. Done